src/test/scala/com/ncredinburgh/sonar/scalastyle/testUtils/TestRuleFinderWithTemplates.scala

```diff
@@ -59,7 +59,4 @@ object TestRuleFinderWithTemplates extends RuleFinder {
   }.asJavaCollection
 
   override def find(query: RuleQuery): Rule = ???
```

Comment:
Why this file wasn't deleted?

Reply:
Because the `RuleFinder` is still used in the old quality profile, so we need to keep it here for the LTS series - it will go away in 7.x.

Reply:
Ok.